src/Api.js

```diff
@@ -8,6 +8,7 @@ import DelayRouter from './routers/DelayRouter';
 import HelloWorldRouter from './routers/HelloWorldRouter';
 import PlacesAutocompleteRouter from './routers/PlacesAutocompleteRouter';
 import RouteRouter from './routers/RouteRouter';
+import SearchRouter from './routers/SearchRouter';
 import MultiRouteRouter from './routers/MultiRouteRouter';
```

Comment:
can we make an `index.js` in routers so we can just import all of these from `./routers/`, instead of each of them individually?